NuRadioMC/utilities/earth_attenuation.py

```diff
@@ -23,19 +24,35 @@ def get_weight(theta_nu, pnu, flavors, mode='simple',cross_section_type = 'ghand
         the zenith angle of the neutrino direction (where it came from, i.e., opposite to the direction of propagation)
     pnu: float or array of floats
         the momentum of the neutrino
+    vertex_position: 3-dim array or None (default)
+        the position of the neutrino interaction
     """
     if(mode == 'simple'):
-        return get_simple_weight(theta_nu, pnu,cross_section_type = cross_section_type)
+        return get_simple_weight(theta_nu, pnu, cross_section_type=cross_section_type)
     elif (mode == "core_mantle_crust"):
-        return get_core_mantle_crust_weight(theta_nu, pnu, flavors,cross_section_type = cross_section_type)
+        return get_core_mantle_crust_weight(theta_nu, pnu, flavors, cross_section_type=cross_section_type)
+    elif (mode == "core_mantle_crust2"):
```

Comment:
CAn we use something a bit more descriptive than just adding a '2'?

Reply:
do you have a suggestion? I couldn't find a better naming.
We could rename the old one to 'core_mantle_crust_simple' but I was trying to avoid that for backward compability